stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>In here there are two narratives, all above "Step 1" in high level, after is technical. </p><p> </p><p>Don't worry about this point @<a href="https://github.com/jack89roberts" target="_blank">jack89roberts</a>, is a comment for the publishing step. We want to have two colours in the text, so those not very technically incline can just read the high level story and skip the technical. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Ok great, maybe we should add a sentence in the intro to explain this in the published version?</p>

Reply:
<p>You mention that the vaccination data is only available as summary total for the whole UK. It looks like the dashboard also breaks the vaccinations down by the 4 nations of the UK. Is it worth mentioning that, even if the story doesn't get into those regional/devolved differences?</p>

Reply:
<p>On the split between high level and technical, I might start the technical block with the previous paragraph: "The documentation for the API..."</p>

Reply:
<p>Good spot for the nations data! I wonder if that's been added since I started writing this, or maybe I just missed it before.</p>